src/main/java/org/selenium/driver/Strategy.java

```diff
@@ -0,0 +1,10 @@
+package org.selenium.driver;
+
+import org.openqa.selenium.WebDriver;
+
+public interface Strategy {
+    String path = "src/main/resources/drivers/";
+    String gecko = "webdriver.gecko.driver";
+    String chrome = "webdriver.chrome.driver";
```

Comment:
Such constants shouldn't be a part of interface. 
It is better to move such variable to the properties and read property in the required place of the code.
Or at least move it to constant class.

Reply:
Fix it